index.js

```diff
@@ -348,8 +348,8 @@ class GithubScm extends Scm {
                   + `"git clone --recursive --quiet --progress --branch ${parentBranch} `
                   + '$CONFIG_URL $SD_CONFIG_DIR"; '
                   + `else ${gitWrapper} `
-                  + `"git clone --depth=50 --recursive --quiet --progress --branch ${parentBranch} `
-                  + '$CONFIG_URL $SD_CONFIG_DIR"; fi');
+                  + '"git clone --depth=50 --no-single-branch --recursive --quiet --progress '
```

Comment:
should we make it default or enable behind a config? Can we try it on a big repository  (our internal sports one) and see the performance implications of this ?

Reply:
Yes! Link to my response
https://github.com/screwdriver-cd/guide/pull/248#discussion_r232417720